Makefile

```diff
@@ -12,6 +12,12 @@ include $(config)
 export LDFLAGS = -pthread -lm
 export CFLAGS = -std=c++11 -Wall -O2 -Iinclude -fPIC
 
+ifdef DMLC_CORE_PATH
```

Comment:
When I do this, it is literally checking for DMLC_CORE_PATH being equal to the string "NONE".  Is this the intent?  For example, code below executes unless I have the line DMLC_CORE_PATH=NONE in config.mk.  Is this what is desired?

ifneq ($(DMLC_CORE_PATH), NONE)
  CFLAGS += -I$(DMLC_CORE_PATH)/include
endif